styles/youtube/catppuccin.user.css

```diff
@@ -599,6 +599,23 @@
       border-color: @accent-color;
     }
 
+    .sbsb_a {
+      background-color: @base;
+    }
+
+    .sbsb_c {
+      color: @text;
+    }
+
+    .sbsb_d {
+      background-color: @surface0;
+    }
+
+    .sbdd_b {
+      border-color: @base;
+      background-color: @base;
+    }
+
     .sbdd_c {
       visibility: hidden;
     }
```

Comment:
```suggestion
    .sbsb_a {
      background: var(--yt-spec-raised-background);
    }
    .sbdd_b {
      border-color: var(--yt-spec-raised-background);
      background-color: var(--yt-spec-raised-background);
    }
    .sbpqs_a, .gsfs {
      color: var(--yt-spec-text-primary);
    }
    .sbsb_i {
      color: var(--yt-spec-call-to-action);
    }
    .sbsb_d {
      background-color: var(--yt-spec-additive-background);
    }
    .sbdd_c {
      visibility: hidden;
    }
```

Reply:
In the future you can just press the "Commit suggestion" button at the bottom of my code block above to commit it right away instead of force pushing :)